src/contracts/pods/EigenPod.sol

```diff
@@ -93,6 +93,18 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice This variable tracks any ETH deposited into this contract via the `receive` fallback function
     uint256 public nonBeaconChainETHBalanceWei;
 
+     /// @notice This variable tracks the total amoutn of partial withdrawals claimed via merkle proofs prior to a switch to ZK proofs for claiming partial withdrawals
+    uint64 public totalPartialWithdrawalAmountClaimedGwei;
```

Comment:
This name is word soup. Can we do something shorter?

How about `sumPartialWithdrawalsClaimedGwei`

If we can get something shorter that's dope too